eng/tools/api-doc-preview/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "api-doc-preview",
+  "type": "module",
+  "private": true,
+  "main": "api-doc-preview.js",
+  "bin": { 
+    "api-doc-preview": "cmd/api-doc-preview.js"
+  },
+  "scripts": {
+    "test": "vitest",
+    "test:ci": "vitest run --coverage --reporter=verbose",
+    "prettier": "prettier \"**/*.js\" --check",
+    "prettier:debug": "prettier \"**/*.js\" --check ---log-level debug",
+    "prettier:write": "prettier \"**/*.js\" --write"
+  },
+  "dependencies": {
+    "@azure-tools/specs-shared": "file:../../../.github/shared"
+  },
+  "devDependencies": {
+    "prettier": "~3.5.3",
+    "vitest": "^3.0.2",
+    "@vitest/coverage-v8": "^3.0.2"
+  }
+}
```

Comment:
Does this tool belong in `eng/tools` or should it go somewhere else? Possibly `.github/shared` though it's going to be used in DevOps and not Actions.